src/components/table/JournalEntryTable.tsx

```diff
@@ -1,150 +1,158 @@
 import React from 'react';
-import clsx from 'clsx';
-import { WithStyles, withStyles } from '@mui/styles';
 import moment from 'moment';
 import _ from 'lodash';
 import { JournalEntryModel, JournalEntryModelEntryTypeEnum } from 'generated/tdr';
 import { TableColumnType } from 'reducers/query';
 import { CustomTheme } from '@mui/material/styles';
 
+import { StatusMapItem } from 'components/table/StatusMapTypes';
+import { CheckCircle } from '@mui/icons-material';
+import { Box, useTheme } from '@mui/material';
 import LightTable from './LightTable';
 
-const styles = (theme: CustomTheme) => ({
-  textWrapper: {
-    ...theme.mixins.ellipsis,
-  },
-  statusContainer: {
-    display: 'flex',
-    alignItems: 'center',
-  },
-  statusIcon: {
-    fontSize: '1rem',
-    marginRight: 10,
-  },
-  statusIconSuccess: {
-    color: theme.palette.success.light,
+interface StatusMap {
+  CREATE: StatusMapItem;
+  UPDATE: StatusMapItem;
+  DELETE: StatusMapItem;
+}
+
+const statusMap: StatusMap = {
+  CREATE: {
+    icon: (props, theme) => (
+      <CheckCircle sx={{ ...props.sx, color: theme.palette.success.light }} />
```

Comment:
```suggestion
    icon: (props) => (
      <CheckCircle sx={{ ...props.sx, color:  success.light }} />
```
We can use the color shorthand here in the sx prop